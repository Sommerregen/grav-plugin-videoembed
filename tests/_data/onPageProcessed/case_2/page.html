<h1>Test 2</h1>

<ul>
    <li>Learn about **Grav** by checking out our dedicated site.</li>
    <li>Download **plugins**, **themes**, as well as other Grav **skeleton** packages from the page.</li>
    <li>Check out our to find out the latest goings on in the Grav-verse.</li>
</ul>

<a href="http://youtu.be/Rx5GBrsfXTA?autohide=0">http://youtu.be/Rx5GBrsfXTA?autohide=0</a>

<h3>Edit this Page</h3>