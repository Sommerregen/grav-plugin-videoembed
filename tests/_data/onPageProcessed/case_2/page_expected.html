<h1>Test 2</h1>

<ul>
    <li>Learn about **Grav** by checking out our dedicated site.</li>
    <li>Download **plugins**, **themes**, as well as other Grav **skeleton** packages from the page.</li>
    <li>Check out our to find out the latest goings on in the Grav-verse.</li>
</ul>

<div class="container"><iframe frameborder="0" width="560" src="//youtube.com/embed/Rx5GBrsfXTA?autoplay=1&autohide=0&wmode=opaque"></iframe></div>

<h3>Edit this Page</h3>